File: api/Models/Restaurant.cs
using System;
using System.Collections;
using System.Linq;
using System.Threading.Tasks;
using Microsoft.AspNetCore.Components.Routing;


namespace api.Models
{

    public class Restaurant{
        public int Id { get; set; }

        public string Name { get; set; } = string.Empty;
        public string Type { get; set; } = string.Empty;
        
    
        public string Country { get; set; } = string.Empty;

        public string City { get; set; } = string.Empty;

        public string Address { get; set; } = string.Empty;
        public string Phone {get; set;} = string.Empty;

         public int Price_Rating {get; set;} //range = 1-5

         public List<Comment> Comments { get; set; } = new List<Comment>();

        //public List<Portfolio> Portfolios { get; set; } = new List<Portfolio>();
    }
}

File: angular-app/my-app/src/app/models/restaurant.ts
export interface Restaurant {
    id: number,
    name: string,
    type: string,
    isOpen: boolean,
    country:string,
    city:string,
    address:string,
    phone:string,
    price_Rating:number,
    acceptTerms: boolean,
    image:string,
    comments:Comment[]
    
  } 

  export interface RestaurantType {
    key: number,
    value: string
  }